python/lsst/ap/verify/testPipeline.py

```diff
@@ -334,7 +334,8 @@ class MockAlardLuptonSubtractTask(PipelineTask):
     ConfigClass = AlardLuptonSubtractConfig
     _DefaultName = "notAlardLuptonSubtract"
 
-    def run(self, template, science, sources, finalizedPsfApCorrCatalog=None):
+    def run(self, template, science, sources, finalizedPsfApCorrCatalog=None,
+            calibrationsCatalog=None):
```

Comment:
If you change `calibrationsCatalog` to `visitSummary` on the ip_diffim PR, also change it here.